galley/pkg/crd/validation/validation.go

```diff
@@ -64,6 +73,11 @@ func RunValidation(vc *WebhookParameters, printf, faltaf shared.FormatFn, kubeCo
 		faltaf("cannot create validation webhook service: %v", err)
 	}
 
+	validationProbe = probe.NewProbe()
+	validationProbe.SetAvailable(nil)
+	validationProbe.RegisterProbe(*livenessProbeController, "validationLiveness")
+	validationProbe.RegisterProbe(*readinessProbeController, "validationReadiness")
```

Comment:
You can only register one controller to each probe. You'll need separate probes for readiness vs. liveness.

Reply:
one